src/Symfony/Component/HttpFoundation/BinaryFileResponse.php

```diff
@@ -350,7 +350,7 @@ public static function trustXSendfileTypeHeader()
      *
      * @return BinaryFileResponse
      */
-    public function deleteFileAfterSend($shouldDelete)
+    public function deleteFileAfterSend($shouldDelete = true)
```

Comment:
Still, don't you think we should let this function remains "as it", even for Symfony 4 ?